src/test/java/step2/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,59 @@
+package step2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static step2.StringAddCalculator.getDelimeter;
+import static step2.StringAddCalculator.getStrNumbers;
+
+public class StringAddCalculatorTest {
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAddCalculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
```

Comment:
```suggestion
        int result = StringAddCalculator.splitAndSum(null);
        assertThat(result).isEqualTo(0);

        int emptyResult = StringAddCalculator.splitAndSum("");
        assertThat(emptyResult).isEqualTo(0);
```

검증하는 결과의 변수를 재할당하면서 사용하기 보다는 하나씩 선언하면 어떨까요?
코드가 많아지면 이러한 코드는 가독성이 떨어지고, 예측가능한 코드를 유지하기 힘들다고 생각해요.

물론 지금같은 코드는 문제가 없지만, 습관을 드리는 시간이라고 생각하고 그러한 방향성으로 해보면 좋을듯해요 🤔

Reply:
반복되는 코드를 줄이기 위해 `@NullAndEmptySource` 이러한 어노테이션을 활용해보아도 좋겠네요!